pyqtgraph/functions_qimage.py

```diff
@@ -61,7 +87,13 @@ def _convert_2dlut_to_1dlut(xp, lut):
     return lut, augmented_alpha
 
 
-def _rescale_float_mono(xp, image, levels, lut):
+def _rescale_and_lookup_float(xp, image, levels, lut, *, forceApplyLut):
+    # It is usually more performant to _not_ apply the lut and
+    # instead use it as an Indexed8 ColorTable. This is only
+    # applicable if the lut has <= 256 entries.
+
+    assert (not forceApplyLut) or (lut is not None)
```

Comment:
Same comment about `assert` as earlier.

Reply:
This one makes sense to be an exception.